packages/svelteui-core/src/components/Table/Table.styles.ts

```diff
@@ -0,0 +1,42 @@
+import { createStyles } from '$lib/styles';
+import type { SvelteUINumberSize } from '$lib/styles';
+
+export interface TableStyleParams {
+	withBorder: boolean;
+	striped: boolean;
+	highlightOnHover: boolean;
+	cellPadding: SvelteUINumberSize;
+	textAlign: 'center' | 'left' | 'right';
+}
+
+export const padding = {
+	sm: 4,
+	md: 8,
+	lg: 16
+};
+
+export default createStyles(
+	(theme, { withBorder, highlightOnHover, striped, cellPadding, textAlign }: TableStyleParams) => {
+		const { themeColor } = theme.fn;
+		return {
+			root: {
+				borderCollapse: 'collapse',
+				cursor: highlightOnHover ? 'pointer' : 'default',
+				border: withBorder ? `1px solid ${themeColor('gray', 6)}` : '',
+				borderStyle: withBorder ? '' : 'hidden',
+				'& > * > th, & > * > td': {
+					padding:
+						typeof cellPadding === 'number' ? `${cellPadding}px` : `${padding[`${cellPadding}`]}px`,
+					border: `1px solid ${themeColor('gray', 6)}`,
```

Comment:
To follow the design system of Mantine:

```suggestion
					border: `1px solid ${themeColor('gray', 3)}`,
```

Reply:
Also, it's missing the darkMode. The same of the hover. Normally checking the style in the mantine codebase helps to follow the correct colors: https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Table/Table.styles.ts

```
darkMode: {
    borderColor: themeColor('dark', 4)
}
```